### PR TITLE
Security Updates

### PR DESCRIPTION
**Pull request by losborne**
Add audits (for bundler and yarn) to travis, so we can see automatically if any gems or node_modules become security vulnerabiltiies.

Fix a couple of yarn vulnerabilities that were found.

Pull requests into cqm-models require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1967
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If applicable, the library version number in `package.json` and `cqm-models.gemspec` has been updated
- [x] Cqm-execution fixtures have been updated with the update_cqm_execution_fixtures.sh script inside server-scripts using this branch in the cqm-converter

**Bonnie Reviewer:**

Name: @mayerm94
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Cypress Reviewer:**

Name: @zlister 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ matrix:
       script:
         - bundle exec rake
         - bundle exec rubocop
+        - bundle exec bundle-audit update && bundle exec bundle-audit check
```
Comment (okeefm) [inline .travis.yml]: Why are the two `ruby` entries here different `language`s, rather than just being two entries in the `rvm` field of one `ruby` language? All the other steps look the same otherwise.

Comment (losborne) [reply .travis.yml]: Not sure why, but I agree that 2 entries in the `rvm` field would be better in this case, will update

Comment (losborne) [reply .travis.yml]: After trying it, it doesn't look like it does 2 build/tests for the 2 different ruby versions if you do it that way.
